pod/perlhack.pod

```diff
@@ -172,16 +172,7 @@ L<perlgit>.
 
 =head2 Read access via Git
 
-You will need a copy of Git for your computer.  You can fetch a copy of
```

Comment:
Some of this verbiage could be left in but also maybe isn't necessary in 2021. LGTM either way